File: Matrix/Python/Addition_of_two_square_matrices.py
class Solution:
	def Addition(self, matrixA, matrixB):
		# Code here
		for i in range(len(matrixA)):
            
            for j in range(len(matrixA[i])):
                
                matrixA[i][j] = matrixA[i][j] + matrixB[i][j]

File: Solutions/Java/Addition_of_two_square_matrices.java
class Solution
{
    public void  Addition(int[][] matrixA, int[][] matrixB)
    {
        // code here
        for (int i=0; i<matrixA.length; i++) {
            
            for (int j=0; j<matrixA[i].length; j++) {
                
                matrixA[i][j] = matrixA[i][j] + matrixB[i][j];
            }
        }
    }
}
